lib/idb-helper.js

```diff
@@ -19,123 +19,139 @@ import idb from 'idb';
  * A wrapper to store for an IDB connection to a particular ObjectStore.
  *
  * @private
- * @class
  */
-function IDBHelper(name, version, storeName) {
-  if (name == undefined || version == undefined || storeName == undefined) {
-    throw Error('name, version, storeName must be passed to the constructor.');
+class IDBHelper {
+  constructor(name, version, storeName) {
+    if (name == undefined || version == undefined || storeName == undefined) {
```

Comment:
Strict `===` are being used elsewhere. Is there a reason the IDB helper is preferring `==`?